### DISCUSSION TITLE
build source error

### DISCUSSION THREAD

**Discussion by ihadoop** [apache/doris#20681] [RFC]
when i run build.sh ,then 

![image](https://github.com/apache/doris/assets/5036560/33dd674a-fd53-44ea-a743-4602a0066dd2)

i visit this url  directly ，it show 404 Not Found。

i found this url in vars.sh

`# libhdfs
HADOOP_LIBS_X86_DOWNLOAD="https://github.com/apache/doris-thirdparty/releases/download/hadoop-libs-3.3.4/hadoop_lib_3.3.4-x86.tar.gz"
HADOOP_LIBS_X86_NAME="hadoop_lib_3.3.4-x86.tar.gz"
HADOOP_LIBS_X86_SOURCE="hadoop_lib_3.3.4-x86"
HADOOP_LIBS_X86_MD5SUM="96117450170487f007ffeca5ddf62f7e"`

is it url corrent?

**Comment by yagagagaga** [comment]
1. Make sure update your code to latest.
2. Download the [doris-thirdparty](https://github.com/apache/doris-thirdparty/releases), decompression it and take it to ${DORIS_HOME}/thirdparty.
3. run `build.sh --clean` and `build.sh`.
